pkg/proto/pbgo/core/workloadmeta.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
-// 	protoc        v5.26.1
+// 	protoc-gen-go v1.28.1
```

Comment:
yeah I'm not sure we should touch those files

Reply:
Indeed, thanks! They were regenerated by mistake, I've removed them in my last commit